snow/engine/snowman/getter/getter_test.go

```diff
@@ -138,21 +85,6 @@ func TestFilterAccepted(t *testing.T) {
 		StatusV: choices.Accepted,
 	}}
 
-	vm.CantLastAccepted = false
-	vm.LastAcceptedF = func(context.Context) (ids.ID, error) {
-		return blk1.ID(), nil
-	}
-	vm.GetBlockF = func(_ context.Context, blkID ids.ID) (snowman.Block, error) {
-		require.Equal(blk1.ID(), blkID)
-		return blk1, nil
-	}
```

Comment:
`GetBlock` is currently immediately overwritten below... I have no idea why this was here.